### DISCUSSION TITLE
Can anyone tell me if this project even alive or dead?

### DISCUSSION THREAD

**Discussion by AhsanSarwar0413** [idurar/idurar-erp-crm#1156] [General]
I have been waiting for one PR to merge for like last 3 weeks. But, the maintainer is not responding to anything.

**Comment by reuel88** [comment]
I pretty sure it alive. I see them updating all the time. Depending on your PR it might not be a priority for them

**Comment by AhsanSarwar0413** [comment]
The last PR merged into the project was around 3-4 months ago and major issue was the signup is missing in the develop branch from where we pull our local branch. There has been week since someone made a fix for it. But, the maintainer is not responding to anyone for last 4 months
